### PR TITLE
Standadise Intellij setting with team

### PR DESCRIPTION
**Pull request by JayChenYJ**
code ready to start refactoring for team product.

### CODE REVIEW

**Review by chewwh09 — APPROVED** [review]
LGTM

**Comment by github-amanda** [comment]
looks good!

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ protected Config initConfig(Path configFilePath) {
             initializedConfig = configOptional.orElse(new Config());
         } catch (DataConversionException e) {
             logger.warning("Config file at " + configFilePathUsed + " is not in the correct format. "
-                    + "Using default config properties");
+                + "Using default config properties");
```
Comment (Md-Fazil) [inline src/main/java/seedu/address/MainApp.java:123]: Look good!

**Review by kumsssss — COMMENTED** [review]
LGTM

**Review by Md-Fazil — COMMENTED** [review]
LGTM!

**Review by kumsssss — APPROVED** [review]

**Review by Md-Fazil — APPROVED** [review]

**Review by github-amanda — APPROVED** [review]

**Pull request merged**